src/main/java/com/stripe/model/Account.java

```diff
@@ -17,6 +17,10 @@
 public class Account extends ApiResource implements HasId, MetadataStore<Account> {
   @Getter(onMethod = @__({@Override})) String id;
   String object;
+  String applicationIcon;
```

Comment:
This is not in the docs for me. None of the `application*` are and it's not something we surface in the other bindings. We should remove

Reply:
The `application_*` attributes are in the OpenAPI spec. I can remove them as they don't seem all that useful today, but on the other hand it doesn't cost us anything to keep them and they'll be available if someone ever needs to use them.

Reply:
I'm in favour of removing. I have never heard of those properties before. They seem more for the dashboard than for the API to be honest. If we added them here we'd have to add them to all the other ones and keep maintaining them.

Reply:
Actually, these attributes can be useful for Connect platforms that use destination charges and their users. Retrieving the `py_` charge object on the destination account returns these attributes in the `source` attribute:
```
  "id": "py_1D4SzzJBZTufeQX6VjXfw9sr",
  "object": "charge",
  ...
  "source": {
    "application_name": "Planet Express",
    "application_url": "http://localhost:3000",
    "id": "acct_16bewcKCFFPkgtRh",
    "object": "account"
  },
  ...
```

Unlike most other attributes, they're only returned if they have non-null values, which is why `application_icon` and `application_logo` are missing in the example above.

Are you still in favor of removing them?

Reply:
huh why did we do something like this, that does not make sense. The charge already has `application` as a hash for that. This information should live *there* in that case. I still think we should at least discuss this internally and do this in a separate PR

Reply:
👍